### DISCUSSION TITLE
升级版本失败，遇到错误

### DISCUSSION THREAD

**Discussion by Looong01** [Coolapk-UWP/Coolapk-UWP#155] [Q&A] [answered]
我是先安装的GitHub的release的版本，发现网络连接失败。看了看Issue说是版本比较老了，我就用Issue里一个老哥说的”Get-AppxPackage *Coolapk* | remove-appxpackage“这个命令给卸载了。然去Azure上下载的8.3版本，然后尝试安装的时候发生了错误

应用安装失败，错误消息: 通过 API Logging data because access was denied for file: \\?\C:\Program Files\WindowsApps\Tangent90.CoolapkUWP_2.1.38.0_neutral_split.scale-125_d0s2e6z6qkbn0\AppxManifest.xml, user SID: S-1-5-18 进行的 AppX 部署操作失败，错误为 0x0 (0x80070005)

![image](https://user-images.githubusercontent.com/69568351/201928769-6c8941d4-ea27-4094-bbdf-4b976c95c82d.png)
![image](https://user-images.githubusercontent.com/69568351/201928876-78e9593b-f02e-40f2-b29a-d3638ee2e552.png)

**Comment by wherewhere** [comment, marked as answer]
你直接按照正常方式安装就行了
